tests/test_dump_file.py

```diff
@@ -8,18 +8,16 @@
 
 @pytest.mark.parametrize("sort_ids", [False, True])
 @pytest.mark.parametrize("shuffle_ids", [False, True])
-@pytest.mark.parametrize("use_gzip", [False, True])
-def test_dump_file_min(snap, use_gzip, shuffle_ids, sort_ids, tmp_path):
+@pytest.mark.parametrize("compression_extensions", ["", ".gz", ".zstd"])
+def test_dump_file_min(snap, compression_extensions, shuffle_ids, sort_ids, tmp_path):
     # create file with 2 snapshots with defaults, changing N & step
     snap_2 = lammpsio.Snapshot(snap.N + 2, snap.box, snap.step + 1)
     snaps = [snap, snap_2]
     if shuffle_ids:
         for s in snaps:
             s.id = s.id[::-1]
-    if use_gzip:
-        filename = tmp_path / "atoms.lammpstrj.gz"
-    else:
-        filename = tmp_path / "atoms.lammpstrj"
+
+    filename = tmp_path / f"atoms.lammpstrj{compression_extensions}"
```

Comment:
With the change above, you will need to do this:
```suggestion
    filename = tmp_path / f"atoms.lammpstrj{compression_extension}"
```